docs/quickstart/try.md

```diff
@@ -11,3 +11,6 @@ has_children: false
 {: .no_toc}
 
 {% include_relative includes/try.md %}
+
+Note: For a fully managed lakeFS service with guaranteed SLAs, try https://lakefs.cloud
```

Comment:
```suggestion
Note: For a fully managed lakeFS service with guaranteed SLAs, try [lakeFS cloud](https://lakefs.cloud)
```
It will make lakeFS cloud text to be a link for https://lakefs.cloud